react/react1/week2/todo/src/TodoItem.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+//{ description, deleteItem, id } is from the parent todolist component
+export function TodoItem({ description, deleteItem, id }) {
+  // change the className
+  const [className, setClassName] = useState("unchecked");
+  function changeStyle() {
+    setClassName((prev) => {
+      // prev = "unchecked"?"unchecked":"checked";
+      if (prev === "unchecked") {
+        return "checked";
+      } else {
+        return "unchecked";
+      }
+    });
+  }
+  // call the delete function from parent todolist component
+  const deletetodo = () => deleteItem(id);
+  return (
+    <div>
+      <label htmlFor="item" className={className}>
+        {description}{" "}
```

Comment:
Nitpick: `{" "}` is a little sign that you want a CSS class instead.

Reply:
right!